lib/commands/apps.js

```diff
@@ -0,0 +1,17 @@
+let extensions = {}, commands = {};
+
+/**
+ *  Gets system and installed applications info
```

Comment:
Please format the docstring properly according to jsdoc documentation rules. For example the `@return` section must be present. Check docs for other extansions for examples

Reply:
okay